articles/cli.md

```diff
@@ -0,0 +1,178 @@
+## Bonsai Command-Line Interface (CLI)
+
+Bonsai CLI (Command-Line Interface) allows workflows to be launched from the OS command-line tool. In addition to opening workflows with the default graphic interface, the CLI opens the possibility to use other advanced features, that will be covered ahead, such as "no-editor" mode, initialization of variable's values, and specifying layouts.
+
+
+&nbsp;
+___
+### Installation and basic use
+Bonsai CLI can be called by targeting ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line:
+
+```cmd
+> pathToBonsaiExe\Bonsai.exe
+```
+Alternatively, by adding ```pathToBonsaiExe``` to your System Path, simply:
+
+```cmd
+> Bonsai
+```
+
+This command will open the default Bonsai editor.
+
+___
+&nbsp;
+### Opening a Workflow
+
+To open an existing workflow, ```myWorkflow.bonsai```, use the syntax:
+
+```cmd
+> Bonsai myWorkflow.bonsai
+```
+
+##### Example:
+
+```cmd
+> Bonsai myCameraWorkflow.bonsai
+> Bonsai "C:\Users\User\Desktop\Example\myCameraWorkflow.bonsai"
+```
+___
+&nbsp;
+### Starting a Workflow
+
+Adding additional flags will change the behavior of Bonsai CLI output. For instance, to launch and start a workflow from the command line we add the ```--start``` flag to the previous command:
```

Comment:
Edit for clarity: 

"Adding flags will change the output behavior of Bonsai CLI. For instance, to launch and immediately start a workflow from the command line, we can add the ```--start``` flag:"